x-pack/legacy/plugins/infra/common/http_api/logs/summary.ts

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import * as rt from 'io-ts';
+
+export const LOGS_SUMMARY_PATH = '/api/logs/summary';
+export const LOGS_SUMMARY_HIGHLIGHTS_PATH = '/api/logs/summary/highlights';
+
+// const esDate = rt.union([rt.string, rt.number]);
+
+export const logsSummaryRequestRT = rt.type({
```

Comment:
I'm missing the information about the log source in here. Either we supply the `sourceId` or we use the source information present on the client side and supply the indices and field configuration. For the GraphQL API we chose the former, but this would be a chance to reconsider.